pkg/cdi/oci.go

```diff
@@ -30,14 +30,33 @@ func (h *Hook) toOCI() spec.Hook {
 	}
 }
 
+// Extra OCI mount option to apply to injected mounts.
+type extraOCIMountOption func(*spec.Mount)
+
+// withMountIDMappings adds UID and GID mappings for the given mount.
+func withMountIDMappings(uid, gid []spec.LinuxIDMapping) extraOCIMountOption {
```

Comment:
Does it make sense to have `uid` and `gid` settable by separate options.

Reply:
I am fine with anything. Separate options, combined options, even with simply doing it like this

```golang
func (m *Mount) toOCI(uids, gids []spec.LinuxIDMapping) spec.Mount {
	om := spec.Mount{
		Source:      m.HostPath,
		Destination: m.ContainerPath,
		Options:     m.Options,
		Type:        m.Type,
		UIDMappings: uids,
		GIDMappings: gids,
	}
	return om
```